packages/service/src/service.ts

```diff
@@ -87,8 +87,8 @@ export default class WdioImageComparisonService extends BaseClass {
     }
 
     beforeTest(test: Frameworks.Test) {
-        this.#currentFile = test.file
-        this.#currentFilePath = resolve(dirname(test.file), FOLDERS.DEFAULT.BASE)
+        this.#currentFile = test.filename
```

Comment:
This should work for and Mocha and Jasmine

```suggestion
        this.#currentFile = test.file || test.filename
```